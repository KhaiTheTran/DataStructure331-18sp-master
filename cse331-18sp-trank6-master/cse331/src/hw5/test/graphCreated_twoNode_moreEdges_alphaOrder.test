# Test graphCreated_twoNode_moreEdges_alphaOrder
# Test with two nodes and five edges in backward order

# Create a graph
CreateGraph graphCreated_twoNode_moreEdges_alphaOrder

# Add two nodes in backward order
AddNode graphCreated_twoNode_moreEdges_alphaOrder yy
AddNode graphCreated_twoNode_moreEdges_alphaOrder xx

# Add five edges with backward order
AddEdge graphCreated_twoNode_moreEdges_alphaOrder yy yy yyyy
AddEdge graphCreated_twoNode_moreEdges_alphaOrder xx yy xxyy9
AddEdge graphCreated_twoNode_moreEdges_alphaOrder xx yy xxyy99
AddEdge graphCreated_twoNode_moreEdges_alphaOrder yy xx yyxx
AddEdge graphCreated_twoNode_moreEdges_alphaOrder xx xx xxxx

# Print the nodes in the graph and the outgoing edges from xx,
# and outgoing from yy
ListNodes graphCreated_twoNode_moreEdges_alphaOrder
ListChildren graphCreated_twoNode_moreEdges_alphaOrder xx
ListChildren graphCreated_twoNode_moreEdges_alphaOrder yy