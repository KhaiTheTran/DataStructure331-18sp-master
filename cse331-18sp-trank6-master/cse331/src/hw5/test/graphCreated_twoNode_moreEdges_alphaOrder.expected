# Test graphCreated_twoNode_moreEdges_alphaOrder
# Test with two nodes and five edges in backward order

# Create a graph
created graph graphCreated_twoNode_moreEdges_alphaOrder

# Add two nodes in backward order
added node yy to graphCreated_twoNode_moreEdges_alphaOrder
added node xx to graphCreated_twoNode_moreEdges_alphaOrder

# Add five edges with backward order
added edge yyyy from yy to yy in graphCreated_twoNode_moreEdges_alphaOrder
added edge xxyy9 from xx to yy in graphCreated_twoNode_moreEdges_alphaOrder
added edge xxyy99 from xx to yy in graphCreated_twoNode_moreEdges_alphaOrder
added edge yyxx from yy to xx in graphCreated_twoNode_moreEdges_alphaOrder
added edge xxxx from xx to xx in graphCreated_twoNode_moreEdges_alphaOrder

# Print the nodes in the graph and the outgoing edges from xx,
# and outgoing from yy
graphCreated_twoNode_moreEdges_alphaOrder contains: xx yy
the children of xx in graphCreated_twoNode_moreEdges_alphaOrder are: xx(xxxx) yy(xxyy9) yy(xxyy99)
the children of yy in graphCreated_twoNode_moreEdges_alphaOrder are: xx(yyxx) yy(yyyy)